src/five_moment/five_moment.h

```diff
@@ -192,6 +193,8 @@ std::unique_ptr<FiveMomentApp<dim>> FiveMomentApp<dim>::create_from_parameters(
                                                     t_end,
                                                     n_writeout_frames);
 
+    ext->prepare_extension(input, app->species, gas_gamma);
```

Comment:
create a TODO issue: have `app` expose whatever is appropriate, and pass in a pointer / make this a virtual method.